js/web/test/test-runner.ts

```diff
@@ -54,12 +57,40 @@ async function loadFile(uri: string): Promise<Uint8Array> {
   }
 }
 
-async function loadTensorProto(uriOrData: string|Uint8Array): Promise<Test.NamedTensor> {
+async function loadTensorProto(uriOrData: string|Uint8Array, allowInt64 = false): Promise<Test.NamedTensor> {
   const buf = (typeof uriOrData === 'string') ? await loadFile(uriOrData) : uriOrData;
-  const tensorProto = onnxProto.TensorProto.decode(buf);
-  const tensor = Tensor.fromProto(tensorProto);
+  const tensorProto = onnx.TensorProto.decode(buf);
+
+  let tensor: ort.Tensor;
+
+  // by default, we don't allow (u)int64. this is for backward compatibility.
+  if (allowInt64 && tensorProto && tensorProto.dataType &&
+      ((tensorProto.dataType === onnx.TensorProto.DataType.INT64 ||
+        tensorProto.dataType === onnx.TensorProto.DataType.UINT64))) {
```

Comment:
the old pure-JS webGL backend uses JS protobuf, which utilizes 3rd party library `Long` instead of `bigint` to represent int64. test runner need to work with it so in webGL kernel some (u)int64 are implicitly cast into 32 bit to make it work.